kolibri/utils/options.py

```diff
@@ -359,6 +377,16 @@ def lazy_import_callback_list(value):
 
 
 base_option_spec = {
+    "FileStorage": {
+        "DEFAULT_FILE_STORAGE": {
```

Comment:
I think I'd want to hew closer to the pattern we have for the Cache and Database options here - and offer simple, pre-specified string options that refer to specific backends. If someone really wants to run a custom backend, they can override the settings file and do what they like.

That way, with specific backends in mind, we can then explicitly enumerate the additional things that need to be specified in each case - for example the default `"file_system"` backend value will need a path, if it's a GCS backend then other things might be required (or may be automagically configured in some cases).